### DISCUSSION TITLE
BTicino MatixGo e Home Assistant - quale protocollo radio ?

### DISCUSSION THREAD

**Discussion by edferoci** [indomus/forum#3171] [ZigBee, Z-Wave, Thread, Matter]
Qualcuno ha esperienza nell'interfacciare gli elementi della serie MatixGo con Home Assistant, come ad esempio il "comando tapparella connesso" (codice JW4027C )?

In rete non ci sono né guide né dettagli sul protocollo utilizzato.
Nella scheda tecnica del gateway  (codice JG4510C ) vengono riportati solo questi dettagli

> Tecnologia Radio: (Maglie di rete senza fili)
> Livello di potenza <100mW
> Frequenza di banda da 2,4 a 2,4835 GHz

Potrebbero utilizzare il protocollo Zigbee ?

**Comment by Faermas** [comment]
io ho serie matixgo ma ti sconsiglio altamente di utilizzarli: nel loro sistema sono una bomba ma per sincronizzarli con HA è un casino, io ho il comando entra ed esci e per farlo funzionare ho perso tantissimo tempo.... e non funziona benissimo. Comunque utilizzano rete zigbee e controlla se con z2m sono compatibili. Buona fortuna

**Reply by giovi21** [reply]
Io ho un impianto con serie btcino living now e pensavo di montare i frutti domotici molto simili a quelli riportati da edferoci.
Onestamente però dovendo gestire dei frangisole mi pare abbiano grandi limitazioni.
2 su tutte:
Non possibile triggerare in base a meteo
Non semplice una gestione molto precisa dell'inclinazione delle lamelle da comando fisico.
Da capire possibile integrazione con stazione meteo netatmo.

**Reply by Faermas** [reply]
Allora tramite l'app home + control ci sono delle automazioni che puoi fare tra cui quella che se il tempo cambia in nuvoloso o tempesta si abbassano automaticamente le tapparelle: io ce l'avevo impostato tramite la loro app e funzionava il problema è che lo faceva sempre anche quando eri a casa (a me una volta mi ha chiuso fuori, meno male che avevo messo una tapparella non la toccava e la lasciava aperta). C'è l'integrazione con netatmo ma non so quanto sono "integrati"
Per quello delle lamelle non so come aiutarti in quanto ho solo delle tapparelle classiche. Ripeto è un ottimo prodotto ma sei molto legato e altrettanto limitato con la loro app. È un mese che sono riuscito a collegare il tasto entra ed esci tramite z2m ad HA e già ho impostato 20 automazioni diverse con quel tasto, cosa che con bticino potevi fare solo entra ed esci. Al momento il collegamento del tasto ad HA è funzionante e stabile anche se non riesco ad aggiornarlo.

**Reply by giovi21** [reply]
Grazie mille per la risposta.
Pensa che ho provato a chiedere anche al servizio assistenza di btcino che però mi ha detto che quest'azione non fosse possibile (nel mio caso devo alzarle in caso di vento forte/tempesta), al che mi sono scoraggiato e quindi sto valutando di acquistare altro...le soluzioni sarebbero quindi shelly 2pm per i quali però ho letto di problemi di longevità/qualità e poi usano il protocollo wifi oppure potrebbe essere qualcosa di casa fibaro (quindi zwave) sperando durino di più

**Reply by Faermas** [reply]
E pensa che sono le automazioni consigliate e preimpostate da Bticino 🫠
Io sinceramente ho deciso di passare a shelly successivamente ed ho diversi shelly dentro casa: mi sono trovato bene con i shelly "classici" , con quelli bluetooth "NI" perché sono comodi e funzionanti ma per esempio TUTTI quelli bluetooth che ho installato non mi danno l'entità  lux (o mi dice 0 o un numero fisso)... proverò a contattare l'assistenza e vediamo che mi dicono. I shelly io li ho installati tramite cloud wifi ma puoi anche collegarli tramite MQTT

**Reply by giovi21** [reply]
Ma mi confermi esistere ancora oggi questa possibilità nell'app? In particolare con la possibilità di tirarle su in caso di mal tempo e non il contrario.
Potrei affidarmi ad HA ma francamente dovendo domotizzare solo questo aspetto mi sembrerebbe sproporzionato come investimento in termini di tempo (almeno per adesso).
Cmq ho richiamato btcino per capire se effettivamente hanno integrato la stazione meteo e cosa è possibile fare e mi richiameranno in giornata. Poi ti aggiorno su cosa mi dicono.

**Reply by Faermas** [reply]
![Screenshot_20240828_120846_Control](https://github.com/user-attachments/assets/bdd608f9-6d3e-494f-a5b2-836bcec887b9)
![Screenshot_20240828_120833_Control](https://github.com/user-attachments/assets/2633cfd4-5a42-4a37-91f6-af282e23ebdf)
Questi sono i due screenshot delle automazioni consigliate da Bticino, ed in più puoi anche farne di altre personalizzate. Naturalmente puoi scegliere quello che vuoi (alzare, abbassare, mettere a metà, etc...)

**Comment by edferoci** [comment]
update: in questi giorni ho fatto un piccolo test e Zigbee2MQTT si collega correttamente al comando tapparella, rilevandolo come questo dispositivo: [BTicino K4027C/L4027C/N4027C/NT4027C](https://www.zigbee2mqtt.io/devices/K4027C_L4027C_N4027C_NT4027C.html#bticino-k4027c-l4027c-n4027c-nt4027c)

In laboratorio risponde correttamente ai comandi "alza" ed "abbassa".
Resta da testare la stabilità della connessione e la risposta a tutti i comandi una volta installato completamente

**Reply by giovi21** [reply]
Guardando varie discussioni online (principalmente in inglese) alcuni lamentavano disaccoppiamento soprattutto in caso di blackout e se ho ben inteso anche problemi con la gestione delle posizioni parziali o di status (ad es se è aperta o chiusa).
Altri riportavano di ronzii dei moduli.
Tu hai avuto modo di verificare?
Sai poi dirci che comportamento hanno i comandi fisici? Cioè se schiacci una sola volta vanno completamente su/giù? E se tieni premuto? In teoria da home + control è possibile settare sia il comportamento come tapparella che quello come frangisole.
In quest'ultimo caso il comportamento prevede che a seguito di una pressione continuata si possa ottenere una regolazione dell'inclinazione delle lamelle e non una salita/discesa completa (che si avrebbe invece con una singola pressione).

**Reply by Faermas** [reply]
Hai fatto un test solo con il comando tapparella? Altri tasti? Perché ero curioso di sapere il tasto entra esci come funzionava a te

**Reply by giovi21** [reply]
Io non ho ancora acquistato il kit. Per questo mi sto documentando sulla soluzione migliore.
Proverò a richiamare btcino per capire se l'integrazione con la stazione meteo netatmo che dovrebbe essere disponibile da fine luglio prevede la possibilità di triggerare in base all'anemometro

**Comment by paobrunelli** [comment]
sto provando in laboratorio l'integrazione dello swtch per tapparelle JW 4027C  ("il dispositivo") in HA con ZHA. Il pairing è semplice, ma induce un comportamento diverso del dispositivo, ed è instabile, nel senso che togliendo l'alimentazione, il dispositivo si disconnette. Provo a dare qualche dettaglio sulle prove fatte.

Per avviare il pairing è necessario togliere la capsula di copertura per poter intervenire sul pulsante di reset. la sequenza che ho eseguito è la seguente:
- all'accensione  il dispositivo ha il led acceso in rosso, funziona come due pulsanti, ciascuno attivo fintanto che viene premuto
- premendo il reset per qualche secondo il led lampeggia in verde
-  accendendo il pairing di ZHA, il dispositivo viene riconosciuto facilmente come "Shutter SW with level control da Legrand" con 5 entità: rappresentate in ZHA come in figura:
![image](https://github.com/user-attachments/assets/fcdf1e38-c9be-41b8-91a6-7ec38ef3b11b)
- il led continua a lampeggiare in verde
- il comportamento del dispositivo è cambiato: abilita tre pulsanti: i due laterali  diventano ad un impulso che accende il carico controllato (avvia l'apertura o la chiusura), che resta acceso finche il dispositivo non viene premuto al centro,, come fosse un pulsante assiale, il che spegne l'interruttore attivo. questo nuovo comportamento è coerente con la scheda Lovelace proposta da HA per l'inserimento in plancia dell'entità. Devo capire il comportamento delle altre entità.
- anche premendo il pulsante di reset il led continua a lampeggiare in verde
- togliendo l'alimentazione al dispositivo per qualche decina di secondi alla riaccensione il led del dispositivo è acceso in rosso e in HA il dispositivo non è connesso, e la riconfigurazione fallisce
- il comportamento meccanico del disopsitivo è ancora tre pulsanti
- il reset, premendo il tasto di reset fino a quando fa un lampeggio verse rosso, non modifica lo stato

Infine su github si trova documentazione suk dispositivo Lagrande  (es: https://github.com/zigbeefordomoticz/wiki/blob/master/en-eng/Corner_Legrand-Netatmo.md) ma il comportamento descritto è diverso

**Comment by Permafrost286** [comment]
Ciao a tutti!

Pensavo di usare lo stesso sistema MatixGo per l'automazione delle tapparelle e successivamente integrazione con HA.
Da quello che ho capito leggendo questa discussione non sono totalmente compatibili oppure è cambiato qualcosa in questi mesi?

Grazie!
